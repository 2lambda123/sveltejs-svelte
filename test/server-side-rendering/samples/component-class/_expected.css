.foo.svelte-40y5y5{color:yellow}