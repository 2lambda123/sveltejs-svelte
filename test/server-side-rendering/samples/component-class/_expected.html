<div><p class="foo svelte-40y5y5">i am a widget</p></div>
